x-pack/plugins/viz_editor/public/common/lib/field_config/field_config.ts

```diff
@@ -16,11 +15,14 @@ export function getOperationsForField(field: Field): SelectOperator[] {
   if (field.type === 'date') {
     return ['date_histogram'];
   }
+
   if (field.type === 'number') {
-    return ['count', 'avg', 'sum'];
+    return ['column', 'avg', 'sum'];
```

Comment:
At the moment this resolves to `null` - should it be `terms`? I think the answer depends on whether we want to have a specific terms operation or not

Reply:
I hope we can have a discussion about this today! It's possible that this field_config is not dynamic enough to actually be useful.